cgi-bin/ScieloXML/sci_getrecord.xis

```diff
@@ -8,6 +8,42 @@
 <include>ScieloXML/sci_getdefine.xis</include>
 
 
+<function name="CreateArticleAbstractAllLangsXML" action="replace" split="occ" tag="4001">
+<!-- Generate XML for abstract in available languages -->
+	<!--
+    	v8000: XML ou ausente
+    	v4001: conteudo de abstracts
+    -->
+	<field action="replace"	tag="1000">
+		<pft>if v8000='BODY' or a(v8000) then '^i<![CDATA[^f]]>' fi</pft>
+	</field>

```

Comment:
@rafaelpezzuto troca o trecho `v8000='BODY' or a(v8000)` por `not v702:'.xml'`

Reply:
@rafaelpezzuto também trocar `<field action="import" tag="list">8000</field>` por `<field action="import" tag="list">702</field>`